PyDSS/element_fields.py

```diff
@@ -0,0 +1,119 @@
+ELEMENT_FIELDS = {
+    "Lines": [
+        {
+            "names": ["Currents", "Powers", 'Voltages', 'PhaseLosses'],
+            "options": [
+                "phase_terminal",
+            ],
+        },
+        {
+            "names": ["SeqVoltages", "SeqPowers", 'SeqCurrents', 'VoltagesMagAng', 'CurrentsMagAng'],
+            "options": [
+                "phase_terminal", "mag_ang"
+            ],
+        },
+    ],
+    "Transformer": [
```

Comment:
This should be `Transformers` (plural), correct?

Reply:
done